drkafka/config/doctorkafka.yaml

```diff
@@ -0,0 +1,245 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###################################################################################
+#                                                                                 #
+#  DoctorKafka global settings, including the zookeeper url for kafkastats topic  #
+#                                                                                 #
+###################################################################################
+doctorkafka:
+  # zookeeper quorum for storing doctorkafka metadata
+  zkurl: zookeeper001:2181,zookeeper002:2181,zookeeper003:2181
+  ui:
+    host: localhost
+    port: 8080
+  # ostrich port #
+  ostrich_port: 2052
+  # tsd host and port.
+  tsd_hostport: localhost:18621
```

Comment:
shall we change this to  
     
        tsd:
             host:  localhost
             port: 18621 

to make it consistent with `ui` host and port setting?